recipes-kernel/linux/linux-nilrt-debug_git.bb

```diff
@@ -12,7 +12,8 @@ SRC_URI += "\
 
 # This is the place to overwrite the source AUTOREV from linux-nilrt.inc, if
 # the kernel recipe requires a particular ref.
-#SRCREV = ""
+SRCREV = "378d8fbcc32082ee008a0df967ebd0f41a182877"
+SRCREV_xilinx-zynq = "051c9569fc919a173fbc7a56c75efdbba3b13b8c"
```

Comment:
Wrong hash, nilrt/21.8/4.14 instead of nilrt/22.8/4.14 which is bc0d982eacfb9764302e888ea6d9f3d3bb4a5cc9